### ISSUE TITLE
Example Transformations Notebook

### ISSUE THREAD

**Issue by mibaumgartner** [PhoenixDL/rising#65]
Give example pipelines for different use cases
- [ ] Classification
- [ ] Segmentation
- [ ] Detection with instance segmentation work around

**Comment by justusschock** [comment]
Can we make multiple notebooks for this?

**Comment by mibaumgartner** [comment]
yes of course :) That probably makes sense, because they need different visualisations

**Comment by justusschock** [comment]
We could also think about integrating these visualisations to the utils part of the framework...
